client/src/modules/authorization/components/ResetPassword.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { PureComponent } from 'react';
+import validator from 'validator';
+import { FormattedMessage } from 'react-intl';
+
+class ResetPassword extends PureComponent {
+  state = {
+    email: '',
+    tipVisible: false
+  };
+
+  handleInputChange = event => {
+    const {
+      target: { value }
+    } = event;
+
+    this.setState({ email: value });
+  };
+
+  handleEmailValidation = () => {
+    const { email } = this.state;
+
+    return validator.isEmail(email);
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const isEmailCorrect = this.handleEmailValidation();
+
+    if (isEmailCorrect) {
+      // handle submit here
+      return;
+    }
+
+    this.handleTipVisibility();
+  };
+
+  handleTipVisibility = () => {
+    this.setState({ tipVisible: true });
+    setTimeout(() => this.setState({ tipVisible: false }), 6000);
```

Comment:
do not hide it with setTimeout - just leave it and recheck when form is resubmitted